traffic_ops/install/bin/_postinstall

```diff
@@ -183,6 +183,22 @@ sub generateDbConf {
     return \%todbconf;
 }
 
+sub setCdnConfGoPort {
+    my $cdnConf = shift;
+    if ( exists $cdnConf->{traffic_ops_golang_port} ) {
```

Comment:
I'm confused as to when the port value might be removed. This should always add the port, if it's missing. If the config is regenerated, it might lose the value, but I'm not seeing a code path here that causes the port value to go missing.